R/kenwardroger.R

```diff
@@ -0,0 +1,160 @@
+#' obtain Kenward-Roger adjustment component
+#' @param tmb_data (`mmrm_tmb_data`)\cr produced by [h_mmrm_tmb_data()].
+#' @param theta (`numeric`)\cr numeric vector of the theta estimate.
```

Comment:
```suggestion
#' @param theta (`numeric`)\cr theta estimate.
```
(no need to repeat type since it is in parentheses already)